packages/xo-web/src/xo-app/hub/recipes/recipe-form.js

```diff
@@ -42,6 +42,14 @@ export default decorate([
           [name]: value,
         })
       },
+      toggleStaticIpAddress(__, sip) {
```

Comment:
Remove unnecessary changes.

Reply:
Why is that unnecessary? Don't I need that to update the checkbox value?

Reply:
Sorry, it's my fault.